subsys/disk/Kconfig

```diff
@@ -131,4 +131,23 @@ config DISK_SDHC_VOLUME_NAME
 	  Disk name as per file system naming guidelines.
 
 endif # DISK_ACCESS_SDHC
+
+config DISK_ACCESS_STM32_SDMMC
+	bool "STM32 SDMMC driver"
+	depends on HAS_STM32CUBE
+	select USE_STM32_HAL_SD
+	select USE_STM32_LL_SDMMC
+	help
```

Comment:
Please add default value based on device tree node presence:
```
	default $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_SDMMC))
```
With following defined upper in the same file:
```
DT_COMPAT_ST_STM32_SDMMC := st,stm32-sdmmc
```